xrdp/xrdp.ini

```diff
@@ -25,8 +25,8 @@ crypt_level=high
 ; openssl req -x509 -newkey rsa:2048 -nodes -keyout key.pem -out cert.pem -days 365
 certificate=
 key_file=
-; specify whether SSLv3 should be disabled
-#disableSSLv3=true
+; set SSL protocols (space separated)
```

Comment:
Please document all allowed values here. Also, the `xrdp.ini.5` manual page needs to be updated. It documents disableSSLv3.